cleanlab/classification.py

```diff
@@ -293,8 +300,155 @@ class label.
+        """Returns a vector of probabilities P(true_label=k)
+        for each example in X.
+
+        Parameters
+        ----------
+        X : :obj:`np.array` of shape (n, m)
+          The test data as a feature matrix."""
+
+        return self.clf.predict_proba(*args, **kwargs)
+
+    def score(self, X, y, sample_weight=None):
+        """Returns the clf's score on a test set X with labels y.
+        Uses the model/clf's default scoring function.
+
+        Parameters
+        ----------
+        X : :obj:`np.array` of shape (n, m)
+          The test data as a feature matrix.
+
+        y : :obj:`np.array<int>` of shape (n,) or (n, 1)
+          The test classification labels as an array.
+
+        sample_weight : :obj:`np.array<float>` of shape (n,) or (n, 1)
+          Weights each example when computing the score / accuracy."""
+
+        if hasattr(self.clf, "score"):
+
+            # Check if sample_weight in clf.score(). Not compatible with Python 2.
+            if "sample_weight" in inspect.getfullargspec(self.clf.score).args:
+                return self.clf.score(X, y, sample_weight=sample_weight)
+            else:
+                return self.clf.score(X, y)
+        else:
+            return accuracy_score(
+                y,
+                self.clf.predict(X),
+                sample_weight=sample_weight,
+            )
+
+    def find_label_issues(
+        self,
+        X=None,
+        labels=None,
+        *,
+        pred_probs=None,
+        thresholds=None,
+        noise_matrix=None,
+        inverse_noise_matrix=None,
+        clf_kwargs={},
+    ):
+        """Runs cross-validation to get out-of-sample pred_probs from `clf`
+        and then calls `filter.find_label_issues(labels, pred_probs)` to find label issues.
+        The resulting label issue mask is saved as an attribute of this LearningWithNoisyLabels instance.
+        Kwargs for `filter.find_label_issues` must have already been specified
+        in the initialization of LearningWithNoisyLabels, not here.
+
+        Unlike `filter.find_label_issues`, which requires `pred_probs`,
+        this method only requires a classifier and it can do the cross-validation training for you.
+        Both methods return the same boolean mask that identifies which datapoints have label issues.
+
+        Note: This method computes the label issues from scratch, to access previously-computed label issues
+        from this LearningWithNoisyLabels instance, use `get_label_issues()` instead.
+
+        This is the method called to find label issues inside `LearningWithNoisyLabels.fit()`.
+        For descriptions of the arguments, see `LearningWithNoisyLabels.fit()` documentation.
+
+        Returns
+        -------
+        label_issues_mask : np.array<bool>
+            This method returns a boolean mask for the entire dataset where True represents
+            a label issue and False represents an example that is accurately labeled with high confidence.
```

Comment:
same as previous comment, can indent 2 spaces to match other docstrings

Reply:
fixed